packages/cardhost/app/lib/card-model-for-browser.ts

```diff
@@ -262,7 +261,11 @@ export default class CardModelForBrowser implements CardModel {
   serialize(): ResourceObject<Saved | Unsaved> {
     return serializeResource(
       'card',
-      this.state.type === 'loaded' ? this.state.url : undefined,
+      this.state.type === 'loaded'
+        ? this.state.url
+        : this.state.id
+        ? cardURL({ realm: this.state.realm, id: this.state.id })
+        : undefined,
```

Comment:
This is really unclear

Reply:
sure, i can turn this into if statements instead